### ISSUE TITLE
Box selection copy copies only the first line

### ISSUE THREAD

**Issue by 64kramsystem** [bmaupin/vscode-copy-without-formatting#1]
Hello!

I've noticed that when I attemp to copy a box selection in copy-without-formatting mode, only the first line is copied.

This is a (ugly) video that makes clear what the problem is.

![issue](https://user-images.githubusercontent.com/1595356/74084196-17d5ce00-4a6d-11ea-866c-62c501719a88.gif)

**Comment by bmaupin** [comment]
Great catch! I'll take a look and see if I can come up with a fix.

In the meantime, someone pointed out the other day that vscode has a setting that was added just after I created this that might even be preferable to using this plugin:

```json
  // Controls whether syntax highlighting should be copied into the clipboard.
  "editor.copyWithSyntaxHighlighting": true,
```
(https://code.visualstudio.com/docs/getstarted/settings#_default-settings)

If you set that to `false`, then the default is to copy without formatting, and there's a new command to "Copy with syntax highlighting".

I'm going to add a note to the readme about that too.
